fossunited/templates/macros/event_card.html

```diff
@@ -31,7 +31,7 @@
         <div class="event-card--location-section">
             <div class="event-card--location">
                 <i class="ti ti-map-pin"></i>
-                <span>{{ event.event_location or "To be Announced" | truncate(22, True)}}</span>
+                <span>{{ event.event_location | truncate(25, True) or "To be Announced"}}</span>
```

Comment:
on second thoughts, i don't know if truncated locations will look good in cards. We should have the whole location

Reply:
Some folks are putting up the entire address in cards, and we are unable to control that up - cards would have different proportions then. 

![Screenshot 2024-07-09 at 9 47 48 PM](https://github.com/fossunited/fossunited/assets/60656677/889e57e4-40fa-46cf-a372-4c52f8cf7b8c)

Both the location and the card is clickable - there is more context in the event page too, plus the city name is highlighted - I think that would work.

Reply:
location is not clickable @RiyaMathew-11 and lets keep it that way. Locations being too long makes sense, but let's control the truncate value to be big enough to atleast show Nimhans like address without truncation. Reference: 
![image](https://github.com/fossunited/fossunited/assets/73123690/344ab4e4-0b3b-49a8-ba39-b9fe2cd60b82)

Reply:
I will update the truncate value - tho why can it not be clickable ? Just noticed that it isn't clickable

Reply:
Ahh, it seems the PR got merged before update - will fix it in coming